doc/api/report.md

```diff
@@ -62,6 +62,80 @@ is provided below for reference.
     "osRelease": "3.10.0-862.el7.x86_64",
     "osVersion": "#1 SMP Wed Mar 21 18:14:51 EDT 2018",
     "osMachine": "x86_64",
+    "osCpus": [
+      {
+        "model": "Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz",
+        "speed": 2700,
+        "user": 88902660,
+        "nice": 0,
+        "sys": 50902570,
+        "idle": 241732220,
+        "irq": 0
+      },
+      {
+        "model": "Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz",
+        "speed": 2700,
+        "user": 88902660,
+        "nice": 0,
+        "sys": 50902570,
+        "idle": 241732220,
+        "irq": 0
+      },
+      {
```

Comment:
perhaps the example would be less verbose and still sufficiently descriptive with only one or two cores? I guess we don't doc the format, other than by example, so the docs aren't expected to link to uv_get_cpu_info (or its node equiv)?

Reply:
sure